core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -642,9 +642,11 @@ class ControllerApis(
       def createResponseCallback(requestThrottleMs: Int,
                                  e: Throwable): UnregisterBrokerResponse = {
         if (e != null) {
+          val errors = Errors.forException(e)
```

Comment:
Could you please leverage `decommissionRequest.getErrorResponse(requestThrottleMs, e)`?
```java
        if (e != null) {
          decommissionRequest.getErrorResponse(requestThrottleMs, e)
        } else {
          new UnregisterBrokerResponse(new UnregisterBrokerResponseData().
            setThrottleTimeMs(requestThrottleMs))
        }
```